ui/addons-store.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: documentation
+title: Add-ons Store
+---
+
+# Add-ons Store
+
+This is the section that allows openHAB to be extended via add-ons.
+There is a huge variety of extension possibilities that openHAB provides:
+
+- **Bindings** are used to connect and control hardware and online services. This section provides
+  - **openHAB Distribution**: official bindings maintained by the openHAB project and
+  - **Community Marketplace**: bindings independently released by the community.
+- **Automation** contains scripting languages, templates and module types
+  - **languages** like Groovy, JRuby, Jython, Java Rules (JRule), JScripting (Nashorn) or
```

Comment:
Not sure if we should list Java rules (JRule), as it is not distribution but community.